packages/instant/src/util/error_reporter.ts

```diff
@@ -0,0 +1,62 @@
+import { logUtils } from '@0x/utils';
+import * as _ from 'lodash';
+
+import { EMBEDDED_DOMAINS, INSTANT_ENVIRONMENT, ROLLBAR_CLIENT_TOKEN, ROLLBAR_ENABLED } from '../constants';
+
+// Import version of Rollbar designed for embedded components
+// See https://docs.rollbar.com/docs/using-rollbarjs-inside-an-embedded-component
+// tslint:disable-next-line:no-var-requires
+const Rollbar = require('rollbar/dist/rollbar.noconflict.umd');
+
+let rollbar: any;
+// Configures rollbar and sets up error catching
+export const setupRollbar = (): any => {
```

Comment:
I used to have this logic defined at the top of the file, outside of a function. My intention was to have it loaded "on import", as you suggested.

*But*, this is what I believe was happening: because the compiler tries to be smart about not import unused code, the code was not executed unless I called a function in the file.  Thus, I created a function called `setupRollbar` which just returned the rollbar instance, forcing the code to be executed.  This was confusing for the reader though, per your comment: ["Also this doesn't appear to be doing anything?"](https://github.com/0xProject/0x-monorepo/pull/1295#discussion_r235429735).  

So, I decided to put all this instantiation logic into the setup function,  this change happened in https://github.com/0xProject/0x-monorepo/pull/1295/commits/22a31246622e7139185d080cae9bef275087e245